### DISCUSSION TITLE
Support multilanguage for AM/PM

### DISCUSSION THREAD

**Discussion by voquanghoa** [quasarframework/quasar#12894] [Ideas / Proposals]
I tried to change the local to replace AM/PM by the works in my language but it didn't work.
Then I discovered that the text AM/PM is hardcoded and there is no proper way to change it. 

![image](https://user-images.githubusercontent.com/10450325/159534913-3d21c402-b6eb-4b5b-8ad8-b6ab1d36a510.png)

My idea is to add the text AM/PM to the multi-languages so that it can be changed by changing languages.

Is it possible? 
Thanks.

**Comment by rstoenescu** [comment]
Isn't this the same in all languages?

**Reply by voquanghoa** [reply]
I know Vietnamese (my native language) and Chinese. Both two languages don't use AM/PM.

So, Instead of saying AM, we say "Sáng" or SA (abbreviation), and PM is Chiều (abbreviation: CH) https://en.wikipedia.org/wiki/Date_and_time_notation_in_Vietnam


Similar for Chinese https://ling-lingchinese.com/2018/11/02/am-and-pm-in-chinese/
https://en.wikipedia.org/wiki/Date_and_time_notation_in_Asia#Greater_China

But anyway, after reading some documents and a little thinking, it's quite more complicated than I thought before :D
